db/queries.py

```diff
@@ -3,44 +3,64 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(Message.objects.filter(text__icontains=word))
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return list(Chat.objects.filter(title__startswith="Untitled"))
 
 
 def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+    return list(User.objects.filter(message__sent__year=2015
+                                    ).values_list("first_name", "last_name"))
 
 
 def get_actual_chats() -> list[Chat]:
-    pass
+    return list(Chat.objects.filter(message__sent__year__gte=2020).distinct())
 
 
 def get_messages_contain_authors_first_name():
-    pass
+    return Message.objects.filter(text__contains=F("user__first_name"))
 
 
 def get_users_who_sent_messages_starts_with_m_or_a() -> list[User]:
-    pass
+    return list(User.objects.filter(
+        Q(message__text__istartswith="a") | Q(message__text__istartswith="m")))
 
 
 def get_delivered_or_admin_messages() -> list[Message]:
-    pass
+    return list(Message.objects.filter(
+        Q(is_delivered=1) | Q(user__username__startswith="admin")))
 
 
 def get_count_messages_sent_by_first_name(first_name: str) -> int:
-    pass
+    messages = Message.objects.filter(user__first_name=first_name)\
+        .aggregate(count_messages=Count("chat"))
+    return messages["count_messages"]
```

Comment:
There is a great function count()
```suggestion
    messages = Message.objects.filter(user__first_name=first_name).count()
```